network-mux/test/Test/Mux.hs

```diff
@@ -101,7 +101,7 @@ smallMiniProtocolLimit = 16*1024
 
 activeTracer :: forall m a. (MonadSay m, Show a) => Tracer m a
 activeTracer = nullTracer
---activeTracer = showTracing sayTracer
+--activeTracer = showTracing _sayTracer
```

Comment:
no need to change this line ;)

Reply:
Agreed, but it is changed to match the current name of the debug function.

Reply:
hmm, I don't like this upstream change: `_names` are reserved for not used identifiers.

Reply:
The function is called `_sayTracer` and it is unused since active tracer is commented out.
But ifdefs is another alternative...